"PRIx64" compile error on Ubuntu 20

**Describe the bug** 
Compiling openthread libraries and examples fails with the following error:
```
/openthread/examples/../src/cli/cli_joiner.cpp: In member function 'otError ot::Cli::Joiner::ProcessDiscerner(uint8_t, char**)':
/openthread/examples/../src/cli/cli_joiner.cpp:76:38: error: expected ')' before 'PRIx64'
   76 |         mInterpreter.OutputLine("0x%" PRIx64 "/%u", discerner->mValue, discerner->mLength);
      |                                ~     ^~~~~~~
      |                                      )
  CXX      libopenthread_cli_ftd_a-cli_network_data.o
```

It appears to be an issue with the PRIx64 macro and inttypes.h/cinttypes. Replacing the define with `<cinttypes>`, as well as defining the macro `__STDC_FORMAT_MACROS` as mentioned [here ](https://github.com/Microsoft/rDSN/issues/197) did not fix this issue.

**To Reproduce**

1. Git commit id: **0d8a99b**
2. IEEE 802.15.4 hardware platform: **NRF52840**
3. Build steps:
I am building on a newly created Ubuntu 20 VM, using these commands:
```
./script/bootstrap
./bootstrap
make -f examples/Makefile-nrf52840 BORDER_AGENT=1 BORDER_ROUTER=1 COAP=1 COMMISSIONER=1 DISABLE_BUILTIN_MBEDTLS=1 DNS_CLIENT=1 DIAGNOSTIC=1 EXTERNAL_HEAP=1 JOINER=1 LINK_RAW=1 MAC_FILTER=1 MTD_NETDIAG=1 SERVICE=1 UDP_FORWARD=1 ECDSA=1 SNTP_CLIENT=1 COAPS=1 DHCP6_SERVER=1 DHCP6_CLIENT=1
```
4. Network topology: N/A

**Expected behavior** 
I would expect the build to complete successfully.

Hi @jwhui, I just tried to compile on the same VM as before, using the branch that #5611 is based on, and am still having a compile error. I fetched and checked out the `cli` branch, reran the boostrap scripts, made clean, and rebuilt with the same make command as before.

The error is different however, and is complaining that `<cinttypes>` is not defined. Is there some additional package for gcc/g++ that I must install?

```
~/openthread/examples/../src/cli/cli_joiner.cpp:36:10: fatal error: cinttypes: No such file or directory
   36 | #include <cinttypes>
```

I also tried this branch on my main development machine, which is using the same 9.2.1 gcc/g++ compilers and am still getting the same issue with `PRIx64`.

@lord-potato Oh oops. I totally forgot that. Thanks for the reminder! 😄 

I just tried again with `JOINER=1` and with @jwhui 's changes in #5611 and got the error @nealjack  is getting 
```
~/openthread/examples/../src/cli/cli_joiner.cpp:36:10: fatal error: cinttypes: No such file or directory
   36 | #include <cinttypes>
```

I'm getting the same error with #5613 in a fresh ubuntu docker container
```
/ot/examples/../src/cli/cli_joiner.cpp: In member function 'otError ot::Cli::Joiner::ProcessDiscerner(uint8_t, char**)':
/ot/examples/../src/cli/cli_joiner.cpp:76:38: error: expected ')' before 'PRIx64'
   76 |         mInterpreter.OutputLine("0x%" PRIx64 "/%u", discerner->mValue, discerner->mLength);
      |                                ~     ^~~~~~~
      |                                      )
```

--- a/script/bootstrap
+++ b/script/bootstrap
@@ -44,20 +44,16 @@ install_packages_apt()
     echo 'Installing GNU Arm Embedded Toolchain...'
 
     PLATFORM=$(lsb_release -is)
-    RELEASE=$(lsb_release -rs)
-    UBUNTU2004=20.04
 
     if [ "$PLATFORM" = "Raspbian" ]; then
         sudo apt-get --no-install-recommends install -y binutils-arm-none-eabi gcc-arm-none-eabi gdb-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
-    elif [ "$PLATFORM" = "Ubuntu" ] && [[ ! $RELEASE < $UBUNTU2004 ]]; then
-        echo "Ubuntu Release >= $UBUNTU2004"
-        # no need to use ppa
-        sudo apt-get --no-install-recommends install -y gcc-arm-none-eabi gdb-multiarch libnewlib-arm-none-eabi
-    elif ! command -v arm-none-eabi-g++; then
-        # add gcc-arm-embedded ppa
-        sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa -y
-        sudo apt-get update
-        sudo apt-get --no-install-recommends install -y gcc-arm-embedded
+    elif [ "$PLATFORM" = "Ubuntu" ]; then
+        sudo apt-get --no-install-recommends install -y ca-certificates wget
+        (cd /opt \
+            && wget -c https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2 \
+            && tar xjf gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2 \
+            && rm gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2 \
+            && ln -s /opt/gcc-arm-none-eabi-9-2020-q2-update/bin/* /usr/local/bin/.)
     fi
 
     echo 'Installing pretty tools useful for code contributions...'

